### ISSUE TITLE
Step 4 

### ISSUE THREAD

**Issue by lucyrose93** [foundersandcoders/oauth-workshop#7]
Advice in Step 4: 

"Note: You only need to pass in 2 of the parameters: the client_id & redirect_uri"

We (inc @oliverjam ) found you only need to pass in 1 parameter. But I may be mistaken.

**Comment by oliverjam** [comment]
Yeah it defaults to using the `callback_url` that you provide when you set-up the application on Github. I'm not sure why you'd ever want to set a different `redirect_uri` here?

**Comment by jsms90** [comment]
Good point guys! Thanks :)
